### ISSUE TITLE
global import for typescript

### ISSUE THREAD

**Issue by Akimotorakiyu** [MikeMcl/bignumber.js#230]
bignumber.js in TheoremJS with typescript

https://github.com/arguiot/TheoremJS/issues/25

I need import bignumber.js globally for browser, import bignumber.js as a module  for node.
![图片](https://user-images.githubusercontent.com/31165554/55702738-15988280-5a0a-11e9-96cc-c5d3c969a4fb.png)

In this case, for browser ,<reference types="bignumber.js" /> need a [global kind](https://www.tslang.cn/docs/handbook/declaration-files/templates/global-d-ts.html) .d.ts file.
![图片](https://user-images.githubusercontent.com/31165554/55702597-a753c000-5a09-11e9-8273-3f567f89dc4d.png)
and then it works
![图片](https://user-images.githubusercontent.com/31165554/55703765-167ee380-5a0d-11e9-9135-302af304d00f.png)

But bignumber.js only has a [module kind](https://www.tslang.cn/docs/handbook/declaration-files/templates/module-d-ts.html) .d.ts file.
![图片](https://user-images.githubusercontent.com/31165554/55702556-87bc9780-5a09-11e9-8511-8b129ebeb595.png)

**Comment by SilverLeaves** [comment]
maybe a [module-class](https://www.tslang.cn/docs/handbook/declaration-files/templates/module-class-d-ts.html) kind .d.ts file is better.

**Comment by SilverLeaves** [comment]
may  a  [module-class](https://www.tslang.cn/docs/handbook/declaration-files/templates/module-class-d-ts.html) kind .d.ts file is better

**Comment by MikeMcl** [comment]
Yes, previously the .d.ts file did include global typings, see #143.

I'll consider creating a separate global typings file, as I did with [decimal.js](https://github.com/MikeMcl/decimal.js).

```js
declare global {
  const BigNumber: BigNumberConstructor;
  type BigNumber = BigNumberInstance;
  namespace BigNumber {
    type Config = BigNumberConfig;
    type Constructor = BigNumberConstructor;
    type Format = BigNumberFormat;
    type Instance = BigNumberInstance;
    type ModuloMode = BigNumberModuloMode;
    type RoundingMode = BigNumberRoundingMode;
    type Value = BigNumberValue;
  }
}
 ```
